content/client-lib-development-guide/features.textile

```diff
@@ -1361,15 +1370,42 @@ h4. ChannelOccupancy
 
 h4. ChannelMetrics
 
-* @(CHM1)@ @ChannelMetrics@ is a type that contains the count of publishers and subscribers, connections and presenceConnections, presenceMembers and presenceSubscribers 
+* @(CHM1)@ @ChannelMetrics@ is a type that contains the count of publishers and subscribers, connections and presenceConnections, presenceMembers and presenceSubscribers
 * @(CHM2)@ The attributes of @ChannelMetrics@ consist of:
 ** @(CHM2a)@ @connections@ integer - the total number of connections to the channel
 ** @(CHM2b)@ @presenceConnections@ integer - the total number of presence connections to the channel
 ** @(CHM2c)@ @presenceMembers@ integer - the total number of presence members for the channel
 ** @(CHM2d)@ @presenceSubscribers@ integer - the total number of presence subscribers for the channel
-** @(CHM2e)@ @publishers@ integer - the total number of publishers to the channel 
+** @(CHM2e)@ @publishers@ integer - the total number of publishers to the channel
 ** @(CHM2f)@ @subscribers@ integer - the total number of subscribers to the channel
 
+h4. BatchResult
+* @(BPA1)@ Contains the results from the batch operation
+* @(BPA2)@ @BatchResult@ has the following attributes:
+** @(BPA2a)@ @responses@ is an array of batch response objects. undefined if the request failed completely (e.g. for an invalid key)
```

Comment:
This sentence doesn't start with a capital letter so maybe it's been cut off? `undefined` also isn't a type in some programming languages (like Go) maybe this description can be more programming language neutral?

Reply:
'responses' is lowercase because it references the field in the IDL of the same name. I can see your point on the use of undefined, although I was using it in the literally sense of "this value is undefined" rather than referencing a language specific concept. I can see how this could be confusing, though, so I'm open to changing it to 'null' (or 'not set' to be fully language agnostic) if you think that would be clearer.

Reply:
`null` is used a lot throughout the spec so I'd vote we go with that.

Reply:
[Done](https://github.com/ably/docs/pull/1421/commits/1e1ded87b7b3c1cc7219e46f0aac1828736be373)